src/vmware/azext_vmware/aaz/latest/vmware/addon/_create.py

```diff
@@ -44,15 +44,15 @@ def _build_arguments_schema(cls, *args, **kwargs):
             help="Name of the addon for the private cloud",
             required=True,
             fmt=AAZStrArgFormat(
-                pattern="^[-\w\._]+$",
+                pattern="^[-\\w\\._]+$",
```

Comment:
Can you create a PR with just the bug fix and not all of these additional changes?

Reply:
i believe the additional changes for the regex and readonly properties are there because i used a newer version of the aaz-dev-tools to generate the client. i can't go back to the old aaz-dev-tools version since it doesn't support python 3.12. i tried regenerating with the earliest version that supports python 3.12 and still see the additional changes